frontend/src/pages/Login/Login.tsx

```diff
@@ -107,13 +107,21 @@ export class Login extends React.Component<LoginProps, LoginState> {
         if (token && user) {
           this.storeTokenPlusUser(token, user);
           this.props.history.push('/');
+          this.savePreference(token);
         } else if (detail) {
           this.setState({ alert: detail });
         }
       })
       .catch(console.log); // tslint:disable-line
   };
 
+  private savePreference(token: string) {
+    getPreference(token)
+      .then((response) => response.json())
+      .then(({ results }) => {
+        localForage.setItem(localForageKeys.PREFERENCES, results[0].preferences);
```

Comment:
You're diving right into `results[0].preferences`. This assumes there will always be results returned. It's safer to check.
```suggestion
        if (results.length) {
            localForage.setItem(localForageKeys.PREFERENCES, results[0].preferences);
        }
```